src/PosthogAnalytics.ts

```diff
@@ -124,9 +125,7 @@ export class PosthogAnalytics {
         const { analyticsID } = getUrlParams();
         // if the embedding platform (element web) already got approval to communicating with posthog
         // element call can also send events to posthog
-        if (analyticsID) {
-          setSetting("opt-in-analytics", true);
-        }
+        setSetting("opt-in-analytics", !!analyticsID);
```

Comment:
I think `Boolean(analyticsID)` is favoured (wonder if there's a lint rule for this).

Reply:
Okay good to know. A lint rule would be great. `!!` is still used in a couple of occasions (I checked before using it).